### PR TITLE
build(js): Upgrade JS SDK to 6.18.1-beta.0

### PR DESCRIPTION
**Pull request by AbhiPrasad**
https://github.com/getsentry/sentry-javascript/pull/4531

This will also fix the console array problem.

### CODE REVIEW

**Comment by evanpurkhiser** [comment]
oh yes yes

**Review by evanpurkhiser — APPROVED** [review]

**Comment by AbhiPrasad** [comment]
Sorry this took a while! 

We had some stuff (https://github.com/getsentry/sentry-javascript/pull/4619, https://github.com/getsentry/sentry-javascript/pull/4531#issuecomment-1055829742) getting in the way :P

File: .github/workflows/scripts/monitor-typescript.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@
 // eslint-disable-next-line
 import * as fs from 'fs';
 import * as Sentry from '@sentry/node';
+import '@sentry/tracing';
```
Comment (evanpurkhiser) [inline .github/workflows/scripts/monitor-typescript.ts:6]: Usually the linter makes you put this in a group above these other things no?

Comment (AbhiPrasad) [reply .github/workflows/scripts/monitor-typescript.ts:6]: Is it because it is under `.github/workflows/scripts/monitor-typescript.ts`? I can move it.

Comment (evanpurkhiser) [reply .github/workflows/scripts/monitor-typescript.ts]: yeah maybe linter isn't checking this?

**Pull request merged**